velox/functions/prestosql/tests/ProbabilityTest.cpp

```diff
@@ -26,6 +26,9 @@ constexpr double kInf = std::numeric_limits<double>::infinity();
 constexpr double kNan = std::numeric_limits<double>::quiet_NaN();
 constexpr double kDoubleMax = std::numeric_limits<double>::max();
 constexpr double kDoubleMin = std::numeric_limits<double>::min();
+constexpr int64_t kIntInf = std::numeric_limits<int64_t>::max();
```

Comment:
@aditi-pandit No, the value returned by std::numeric_limits<int64_t>::infinity() is 0 and I could not use double infinity since my parameters are int64. So std::numeric_limits<int64_t>::max() is used as it returns a huge number (9223372036854775807).

Reply:
Hmm.... Then in that case there isn't really a distinction between the tests with kIntInf and kIntMax. Lets just leave the kIntMax tests around then.